The error is not accurately describe

Very nice library! But there is one drawback.

Run this test
```go
package baloo

import (
	"gopkg.in/h2non/baloo.v3"
	"testing"
)

func Test(t *testing.T) {
	b := baloo.New("http://www.ya.ru")
	_ = b.Get(".").Expect(t).Status(11).Done()
	_ = b.Get(".").Expect(t).Status(11).Done()
	_ = b.Get(".").Expect(t).Status(11).Done()
}
```

Look result
```
=== RUN   Test
--- FAIL: Test (0.35s)
    expect.go:204: Unexpected status code: 404 != 11
    expect.go:204: Unexpected status code: 404 != 11
    expect.go:204: Unexpected status code: 404 != 11
FAIL

Process finished with exit code 1
```

There is no way to know exactly where the test failed. The error message point inside the library

--- a/assert/status.go
+++ b/assert/status.go
@@ -39,7 +39,7 @@ func StatusRedirect() Func {
 func StatusEqual(code int) Func {
 	return func(res *http.Response, req *http.Request) error {
 		if res.StatusCode != code {
-			return fmt.Errorf("Unexpected status code for: %s %s => %d != %d", req.Method, req.Url.String(), res.StatusCode, code)
+			return fmt.Errorf("Unexpected status code for: %s %s => %d != %d", req.Method, req.URL.String(), res.StatusCode, code)
 		}
 		return nil
 	}

--- a/expect.go
+++ b/expect.go
@@ -3,6 +3,8 @@ package baloo
 import (
 	"fmt"
 	"net/http"
+	"path"
+	"runtime"
 
 	"gopkg.in/h2non/baloo.v3/assert"
 	"gopkg.in/h2non/gentleman.v2"
@@ -27,6 +29,8 @@ func FlushAssertFuncs() {
 // TestingT implements part of the same interface as testing.T
 type TestingT interface {
 	Error(args ...interface{})
+	Fail()
+	Logf(format string, args ...interface{})
 }
 
 // Expect represents the HTTP expectation suite who is
@@ -201,7 +205,13 @@ func (e *Expect) Done() error {
 	// Run assertions
 	err = e.run(res.RawResponse, res.RawRequest)
 	if err != nil {
-		e.test.Error(err)
+		_, fileName, line, ok := runtime.Caller(1)
+		if !ok {
+			//fallback to test.Error
+			e.test.Error(err)
+		}
+		e.test.Logf("%s:%d: %s\n", path.Base(fileName), line, err)
+		e.test.Fail()
 	}
 
 	return err

